### ISSUE TITLE
Accessing/updating progressbar when multiple uploads

### ISSUE THREAD

**Issue by devtrends** [LPology/Simple-Ajax-Uploader#14]
I am creating some mark-up for each file to upload in onSubmit. Part of this mark-up (but not all) is the progress bar. I then call this.setProgressBar on the progress bar to get the nice progress display that your code handles. In the onComplete event, I want to remove the progress bar and replace it with some other mark-up, whilst retaining the rest of the mark-up that was added in onSubmit.

If I have a single file selection/upload, this is easy but when allow multiple selections, I need some way of determining which bit of mark-up to update since there will be one per upload. Initially I thought I could just use filename and store it as data but this fails when multiple uploads have the same filename which whilst being very rare for desktop clients, happens every time for IOS because all uploads are called image.jpg.

I saw some uid stuff in the JS but before I delve deeper, wondered if you had any ideas?

Basically, all I need is a way to sync the onSubmit and onComplete events without relying on filename which is not unique for IOS.

Thx for a great library.
